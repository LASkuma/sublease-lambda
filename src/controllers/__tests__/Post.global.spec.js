jest.mock('uuid-random', () => () => 1);

const LeaseFactory = require('../../models/Lease').default;
const Post = require('../Post').default;

describe('Post controller', () => {
  it('should insert id, user, createdAt, status, etc.', async () => {
    LeaseFactory.create = jest.fn().mockResolvedValue({});
    Date.now = jest.fn().mockReturnValue(100000);

    expect.assertions(1);

    const result = await Post.create({}, { id: 101 });
    expect(result).toMatchSnapshot();
  });
});
