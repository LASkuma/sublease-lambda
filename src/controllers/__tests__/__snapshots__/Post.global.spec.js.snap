// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post controller should insert id, user, createdAt, status, etc. 1`] = `
Object {
  "createdAt": 100000,
  "id": 1,
  "status": "Active",
  "updatedAt": 100000,
  "user": 1,
}
`;
